src/In.ProjectEKA.DefaultHip/DataFlow/Collect.cs

```diff
@@ -38,17 +41,28 @@ public async Task<Option<Entries>> CollectData(DataRequest dataRequest)
 
         private bool WithinRange(HiDataRange range, DateTime date)
         {
-            var fromDate = DateTime.ParseExact(range.From, "yyyy-MM-dd", null);
-            var toDate = DateTime.ParseExact(range.To, "yyyy-MM-dd", null);
+            var fromDate = ParseDate(range.From);
+            var toDate = ParseDate(range.To);
             return date >= fromDate && date < toDate;
         }
 
+        private static DateTime ParseDate(string dateString)
+        {
+            var formatStrings = new string[]
+            {
+                "yyyy-MM-dd", "yyyy-MM-dd hh:mm:ss", "yyyy-MM-dd hh:mm:ss tt", "yyyy-MM-ddTHH:mm:ss.fffzzz", 
+                "dd/MM/yyyy", "dd/MM/yyyy hh:mm:ss", "dd/MM/yyyy hh:mm:ss tt", "dd/MM/yyyyTHH:mm:ss.fffzzz" 
+            };
+            DateTime.TryParseExact(dateString, formatStrings, CultureInfo.CurrentCulture, DateTimeStyles.None,
```

Comment:
What I understood is DateTime.Parse or TryParse works pretty well, https://docs.microsoft.com/en-us/dotnet/api/system.datetime.parse?view=netframework-4.8#the-string-to-parse. 

Whatever you have done is OK.